packages/functional-tests/src/tests/gltf-concurrency-test.ts

```diff
@@ -57,9 +61,10 @@ export default class GltfConcurrencyTest extends Test {
         const bottle = await MRESDK.Actor.CreateFromPrefab(this.app.context, {
             prefabId: bottleAsset.prefabs.byIndex(0).id
         });
+        actors.push(bottle);
 
         await delay(10000);
-        destroyActors([runner, gearbox, bottle]);
+        destroyActors(actors);
```

Comment:
The app handles actor destruction now. You shouldn't have to do this.

Reply:
So I can remove all the cleanup logic?